src/Symfony/Component/Serializer/Encoder/ChainDecoder.php

```diff
@@ -32,6 +32,8 @@ public function __construct(array $decoders = array())
 
     /**
      * {@inheritdoc}
+     * 
+     * @throws RuntimeException If no decoder is found.
```

Comment:
@stof I agree with that, what I'm disagreeing with is not documenting `@throws \Symfony\Component\Serializer\Exception\Exception` at the interface level, because it's not an implementation detail anymore.